civcraftConfigs/tetconfig.yml

```diff
@@ -82,48 +81,138 @@ monster:
       type: CHICKEN
       spawn_chance: 0.2
       identifier: chicken
-      minimum_light_level: 8
+      minimum_light_level: 7
       blocks_to_spawn_on:
       - DIRT
       - GRASS
+      - SAND
     horse:
       type: HORSE
       spawn_chance: 0.001
       identifier: horse
-      minimum_light_level: 8
+      minimum_light_level: 7
       blocks_to_spawn_on:
       - DIRT
       - GRASS
+      - SAND
     pig:
       type: PIG
       spawn_chance: 0.05
       identifier: pig
-      minimum_light_level: 8
+      minimum_light_level: 7
       blocks_to_spawn_on:
       - DIRT
       - GRASS
+      - SAND
     rabbit:
       type: RABBIT
       spawn_chance: 0.02
       identifier: rabbit
-      minimum_light_level: 8
+      minimum_light_level: 7
       blocks_to_spawn_on:
       - DIRT
       - GRASS
     sheep:
       type: SHEEP
       spawn_chance: 0.04
       identifier: sheep
-      minimum_light_level: 8
+      minimum_light_level: 7
       blocks_to_spawn_on:
       - DIRT
       - GRASS
     ocelots:
       type: OCELOT
       spawn_chance: 0.05
       identifier: ocelot
-      minimum_light_level: 8
-        
+      minimum_light_level: 7
+    JungleSkelly:
+     identifier: jungleskelly
+     type: SKELETON
+     range: 80
+     amount: 1
+     name: §kFive §2JungleSkelly §kFive 
+     on_hit_message: YOU KILLED MY COCO. PISS OFF M8
+     spawn_chance: 0.2
+     drops:
+      PowBone:
+       material: BONE
+       amount: 2
+       lore: Coco Knocker
+       enchants:
+        KB1:
+         enchant: KNOCKBACK
+         level: 1
+     equipment:
+      BluntHotRod:
+       material: BONE
+       enchants:
+        KB8:
+         enchant: KNOCKBACK
+         level: 8
+        S2:
+         enchant: DAMAGE_ALL
+         level: 2
+      firechest451:
+       material: LEATHER_CHESTPLATE
+       enchants:
+        Prot451:
+         enchant: PROTECTION_FIRE
+         level: 451
+      firelegs451:
+       material: LEATHER_LEGGINGS
+       enchants:
+        Prot451:
+         enchant: PROTECTION_FIRE
+         level: 451
+      fireBoots154:
+       material: LEATHER_BOOTS
+       enchants:
+        FeatherFall4:
+         enchant: PROTECTION_FALL
+         level: 3
+#Warning: Glass helmet may be non functional. Eleminate glass helmet if needed.
+     glasshelm:
+       material: GLASS;0
```

Comment:
Yeah that number is the durability, but that's parsing specific notation. If you want to do it here it would have to look like:

material: GLASS
durability: 0